README.md

```diff
@@ -7,201 +7,168 @@
+```bash
+spice run
+```
 
-We will add intelligence to a sample application, **ServerOps**, by creating and training a Spicepod that offers recommendations to the application for different server operations, such as performing server maintenance.
+You should now see the following:
 
-If you are using GitHub Codespaces, skip Step 1. and continue with Step 2., as the repository will already be cloned.
+```bash
+Spice.ai runtime starting...
+Using latest 'local' runtime version.
+2024-02-20T23:54:31.313288Z  INFO runtime::http: Spice Runtime HTTP listening on 127.0.0.1:3000
+2024-02-20T23:54:31.313347Z  INFO runtime::flight: Spice Runtime Flight listening on 127.0.0.1:50051
+2024-02-20T23:54:31.313532Z  INFO runtime::opentelemetry: Spice Runtime OpenTelemetry listening on 127.0.0.1:50052
+2024-02-20T23:54:32.208615Z  INFO runtime: Loaded dataset: eth_recent_logs
+2024-02-20T23:54:32.209376Z  INFO runtime::dataconnector: Refreshing data for eth_recent_logs
+```
 
-Step 1. Clone the Spice.ai quickstarts repository:
+**Step 7** In a seperate terminal window, you can query against the dataset using the spice sql tool:
 
 ```bash
-cd $HOME
-git clone https://github.com/spiceai/quickstarts
-cd quickstarts/serverops
+spice sql
 ```
 
-Step 2. Start the Spice runtime with `spice run`:
+You should now see:
 
 ```bash
-cd $HOME/quickstarts/serverops
-spice run
+Welcome to the interactive Spice.ai SQL Query Utility! Type 'help' for help.
+
+show tables; -- list available tables
 ```
 
-Step. 3. In a new terminal, add the ServerOps quickstart pod:
+Entering `show tables;` should print out the following table:
+
+```
+sql> show tables;
++---------------+--------------------+-----------------+------------+
+| table_catalog | table_schema       | table_name      | table_type |
++---------------+--------------------+-----------------+------------+
+| datafusion    | public             | eth_recent_logs | BASE TABLE |
+| datafusion    | information_schema | tables          | VIEW       |
+| datafusion    | information_schema | views           | VIEW       |
+| datafusion    | information_schema | columns         | VIEW       |
+| datafusion    | information_schema | df_settings     | VIEW       |
++---------------+--------------------+-----------------+------------+
+```
 
-So that we can leave Spice.ai running, add the quickstart pod in a new terminal tab or window. If you are running in GitHub Codespaces, you can open a new terminal by clicking the split-terminal button in VS Code.
+You can enter custom queries:
 
 ```bash
-spice add quickstarts/serverops
+sql> SELECT topics FROM eth_recent_logs LIMIT 5;
 ```
 
-The Spice.ai CLI will download the ServerOps quickstart pod and add the pod manifest to your project at `spicepods/serverops.yaml`.
+### Run through a data demo step by step here. Include a video showing the steps
 
```

Comment:
Transition to Samples

Reply:
One sample is sufficient.

Enumerate 3 other quick start tutorials:
spice.ai login, local dataset configure, ML models.

Reply:
Include one of the streaming datasets.

Reply:
Include DuckDB and Postgres backends